src/App.js

```diff
@@ -9,21 +10,27 @@ import Header from "./components/Header.js";
 import Footer from "./components/Footer.js";
 
 export default function App() {
+  const [sudokuDifficulty, setSudokuDifficulty] = useState("");
```

Comment:
what happens if the user reloads the page while doing the sudoku?
Then this would be set to "" and no sudoku will be fetched. I guess this will be solved when you implement local storage to store the progress, but just so you keep it in mind

Reply:
Yeah it's on my list, thanks for the reminder:)